xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodecSurface.cpp

```diff
@@ -18,202 +18,141 @@
-  if (mci && !mci->IsReleased())
+  if (mci)
   {
-    // this hack is needed to get the 2D mode of a 3D movie going
-    RENDER_STEREO_MODE stereo_mode = g_graphicsContext.GetStereoMode();
-    if (stereo_mode)
-      g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_LEFT);
-
     ManageRenderArea();
-
-    if (stereo_mode)
-      g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_OFF);
-
-    CRect dstRect(m_destRect);
-    CRect srcRect(m_sourceRect);
-    switch (stereo_mode)
-    {
-      case RENDER_STEREO_MODE_SPLIT_HORIZONTAL:
-        dstRect.y2 *= 2.0;
-        srcRect.y2 *= 2.0;
-      break;
-
-      case RENDER_STEREO_MODE_SPLIT_VERTICAL:
-        dstRect.x2 *= 2.0;
-        srcRect.x2 *= 2.0;
-      break;
-
-      case RENDER_STEREO_MODE_MONO:
-        dstRect.y2 = dstRect.y2 * (dstRect.y2 / m_sourceRect.y2);
-        dstRect.x2 = dstRect.x2 * (dstRect.x2 / m_sourceRect.x2);
-      break;
-
-      default:
-      break;
-    }
-
-
-    // Handle orientation
-    switch (m_renderOrientation)
-    {
-      case 90:
-      case 270:
-      {
-        int diffX = 0;
-        int diffY = 0;
-        int centerX = 0;
-        int centerY = 0;
-
-        int newWidth = dstRect.Height(); // new width is old height
-        int newHeight = dstRect.Width(); // new height is old width
-        int diffWidth = newWidth - dstRect.Width(); // difference between old and new width
-        int diffHeight = newHeight - dstRect.Height(); // difference between old and new height
-
-        // if the new width is bigger then the old or
-        // the new height is bigger then the old - we need to scale down
-        if (diffWidth > 0 || diffHeight > 0 )
-        {
-          float aspectRatio = GetAspectRatio();
-          // scale to fit screen width because
-          // the difference in width is bigger then the
-          // difference in height
-          if (diffWidth > diffHeight)
-          {
-            newWidth = dstRect.Width(); // clamp to the width of the old dest rect
-            newHeight *= aspectRatio;
-          }
-          else // scale to fit screen height
-          {
-            newHeight = dstRect.Height(); // clamp to the height of the old dest rect
-            newWidth /= aspectRatio;
-          }
-        }
-
-        // calculate the center point of the view
-        centerX = m_viewRect.x1 + m_viewRect.Width() / 2;
-        centerY = m_viewRect.y1 + m_viewRect.Height() / 2;
-
-        // calculate the number of pixels we need to go in each
-        // x direction from the center point
-        diffX = newWidth / 2;
-        // calculate the number of pixels we need to go in each
-        // y direction from the center point
-        diffY = newHeight / 2;
-
-        dstRect = CRect(centerX - diffX, centerY - diffY, centerX + diffX, centerY + diffY);
-
-        break;
-      }
-
-      default:
-        break;
-    }
-
-    mci->RenderUpdate(srcRect, dstRect);
```

Comment:
see commit message, I still believe that this is handled in Rendermanager::configure change.
If it's not working, let me know pls.